docs/integrations/mirroring.md

```diff
@@ -224,19 +224,22 @@ def get_mapping_fields_command():
+* **dbotMirrorInstance** - the instance through which you will be mirroring.
+- **dbotDirtyFields** for fields we changed locally and therefore will not be updated from the remote.
+- **dbotCurrentDirtyFields** for fields that are going to be sent remotely as a delta in the next iteration of the job - probably should remain invisible to the end user.
+- **dbotMirrorLastSync** timestamp that includes the last time we synched this incident with the remote system.
 * **dbotMirrorTags** - tags for mirrored-out entries (comment/files).
 
 ## Debugging
 
-* **getMirrorStatistics** command - a hidden command that returns mirroring statistics: total mirrored incidents count, rate limit hit count, last run, closed incident records and closed rate limited incident records.
+* **getMirrorStatistics** command - a hidden command that returns mirroring statistics: total mirrored, rate limited, last run, closed incident records and closed rate limited incident records.
 * **getSyncMirrorRecords** - a hidden command that returns records that hold internal mirroring metadata for each mirrored incident.
-* **get-remote-data** - is runnable through the war room if the command is defined in the yml file.
-* **get-modified-remote-data** - is runnable through the war room if the command is defined in the yml file.
-* **get-mapping-fields** - is runnable through the war room with no arguments and see the results.
-* **purgeClosedSyncMirrorRecords** - receives 2 arguments: mandatory `dbotMirrorInstance` for which integration to purge and optional `olderThan` in days. Since this bucket is forever growing (with all the closed incidents that are mirrored), might be a good idea to purge when you're sure that there will be no update from the remote system on the really old incidents.
+* **get-remote-data** - is runnable through the War Room if the command is defined in the yml file.
+* **get-modified-remote-data** - is runnable through the War Room if the command is defined in the yml file.
+* **get-mapping-fields** - is runnable through the War Room with no arguments and displays the results.
+* **purgeClosedSyncMirrorRecords** - receives 2 arguments: mandatory `dbotMirrorInstance` for which integration to purge and optional `olderThan` in days. Since this bucket is forever growing (with all the closed incidents that are mirrored), it might be a good idea to purge when you're sure that there will be no update from the remote system on the really old incidents.
+*  **triggerDebugMirroringRun (Available from 6.6)** - is runnable from the War Room in order to debug a full mirroring run over existing incidents. You can add an incident ID to the command to get information about a specific incident. If no ID is given, the incident will be loaded from the War Room context. The output of this command is a unique debug log file that includes logs for the entire mirroring flow. In addition, this debug log includes the content of the integrations' debug-mode logs.
```

Comment:
provide real example of how to run this command. something like
`!triggerDebugMirroringRun incidentId=100`